blueflood-http/src/integration-test/java/com/rackspacecloud/blueflood/http/HttpIntegrationTestBase.java

```diff
@@ -259,7 +330,7 @@ public URIBuilder getMetricsURIBuilder() throws URISyntaxException {
 
     public URIBuilder getMetricDataQueryURIBuilder() throws URISyntaxException {
         return new URIBuilder().setScheme("http").setHost("127.0.0.1")
-                .setPort(httpPortQuery).setPath("/v2.0/tenantId/metrics/search")
+                .setPort(httpPortQuery).setPath(getSearchPath)
```

Comment:
@izrik this is just a default path to show the format of the path (note the hard coed "tenantId" string in the previous version.  If you look at where getMetricDataQueryURIBuilder() are actually use, they would have a setPath call there to replace it with the correct path (i.e. see queryMetricIncludeEnum()).

This is similar to how the other other getXXXURIBuilder methods in the test class are written -- all i am doing is making the string a variable so it can be reused in getQuerySearchURI.